src/test/java/api/loans/ScheduledDueDateNoticesProcessingTests.java

```diff
@@ -0,0 +1,385 @@
+package api.loans;
+
+import static api.support.matchers.PatronNoticeMatcher.hasEmailNoticeProperties;
+import static api.support.matchers.ScheduledNoticeMatchers.hasScheduledLoanNotice;
+import static java.util.Comparator.comparing;
+import static org.folio.circulation.support.JsonPropertyFetcher.getDateTimeProperty;
+import static org.hamcrest.CoreMatchers.hasItems;
+import static org.hamcrest.Matchers.hasSize;
+import static org.hamcrest.junit.MatcherAssert.assertThat;
+
+import java.net.MalformedURLException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
+
+import org.awaitility.Awaitility;
+import org.folio.circulation.domain.policy.Period;
+import org.folio.circulation.support.http.client.IndividualResource;
+import org.hamcrest.Matcher;
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+import org.junit.Before;
+import org.junit.Test;
+
+import api.support.APITests;
+import api.support.builders.CheckOutByBarcodeRequestBuilder;
+import api.support.builders.NoticeConfigurationBuilder;
+import api.support.builders.NoticePolicyBuilder;
+import api.support.fixtures.NoticeMatchers;
+import io.vertx.core.json.JsonObject;
+
+public class ScheduledDueDateNoticesProcessingTests extends APITests {
+
```

Comment:
Please can this whitespace gap be reduced

Reply:
Done